apps/console/src/features/api-resources/api/api-resources.ts

```diff
@@ -337,3 +339,32 @@ export const deleteScopeFromAPIResource = (
                 error.config);
         });
 };
+
+/**
+ * Get the authorized APIs of the application with authorized permissions.
+ *
+ * @param appId - Application ID.
+ *
+ * @returns A promise containing the response.
+ */
+export const useGetAuthorizedAPIList = <Data = AuthorizedAPIListItemInterface[], Error = RequestErrorInterface>(
+    applicationId: string
+): RequestResultInterface<Data, Error> => {
+    const requestConfig: RequestConfigInterface = {
+        method: HttpMethods.GET,
+        url: `${ store.getState().config.endpoints.applications }/${ applicationId }/authorized-apis`
+    };
+
+    /**
+     * Pass `null` if the `apiResourceId` is not available. This will prevent the request from being called.
+     */
+    const { data, error, isValidating, mutate } = useRequest<Data, Error>(applicationId ? requestConfig : null);
+
+    return {
+        data,
+        error: error,
+        isLoading: !error && !data,
+        isValidating,
+        mutate
+    };
+};
```

Comment:
Move to a new file. `api-resources/api/useGetAuthorizedAPIList`

Reply:
Fixed